pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -66,6 +66,8 @@ const (
 	ZeroedThickDiskType       = "zeroedThick"
 	VolDir                    = "kubevols"
 	RoundTripperDefaultCount  = 3
+	DummyVMName               = "kubernetes-helper-vm"
```

Comment:
We are keeping this Dummy VM around for all the create disk operations. 
If 2 requests for createDisk come at same time, the VC would serialize these operations.